ComputeClient/Compute.Client/ApiUris.cs

```diff
@@ -18,6 +18,9 @@ public static class ApiUris
         /// <summary>	The MCP 2.10 prefix. </summary>
         public const string MCP2_10_PREFIX = "caas/2.10/";
```

Comment:
Do we know if have a seperate task to bump all the API versions to _2_11_ at some stage for zoidberg?

Can't remember when it was done the last time

Reply:
No, I'll go create one. Are all 2.10 apis being bumped to 2.11?

Reply:
I've bumped the version in this PR as part of this commit https://github.com/jimantz/Compute.Api.Client/pull/17/commits/720e2f64ba67b90b3bc76b55dc7ac83444b25c41